components/pages/blog/post/ArticleHeader.jsx

```diff
@@ -129,7 +129,7 @@ const ArticleHeader = ({ frontmatter, author }) => {
         <div className="component-content">
           <div className="category-eyebrow">
             <span className="category-eyebrow__category category_update">
-              UPDATE
+              { frontmatter.category.toUpperCase() }
```

Comment:
We probably don't need `.toUpperCase()` because that should be handled in CSS. Also, this should be tied to a categories data file. If you don't mind, I can do this in my next PR.

Reply:
Sure thing